### DISCUSSION TITLE
Support nulls=stripped introduced by PostgREST 11.2.0

### DISCUSSION THREAD

**Discussion by vrilcode** [supabase/supabase#16479] [Feature Requests]
PostgREST 11.2.0 introduced stripping of null value properties in JSON results for arrays and single object responses. To support that functionality in [postgrest-js](https://github.com/supabase/postgrest-js) a method like `stripNulls()`could be introduced. For instance: `client.from('table').select().stripNulls()`.

Further information:

https://github.com/PostgREST/postgrest/releases/tag/v11.2.0 (see "Resource Representation")
https://postgrest.org/en/stable/references/api/resource_representation.html#stripped-nulls
https://github.com/PostgREST/postgrest/blob/main/test/spec/Feature/Query/NullsStrip.hs

https://github.com/PostgREST/postgrest/issues/1601
https://github.com/PostgREST/postgrest/pull/2894
https://github.com/PostgREST/postgrest/pull/2899

**Comment by steve-chavez** [comment]
@vrilcode Could you open this one on postgrest-js?

**Reply by vrilcode** [reply]
That's what I wanted to do, but when I try to create a "Feature request" issue (https://github.com/supabase/postgrest-js/issues/new/choose), I get redirected to this forum.

**Reply by steve-chavez** [reply]
@vrilcode Just noted that. Opened the issue for you on https://github.com/supabase/postgrest-js/issues/460

**Reply by vrilcode** [reply]
Thanks!
